src/features/LoginView/GoogleLogin.js

```diff
@@ -0,0 +1,52 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { StyleSheet, View, TouchableOpacity, Image } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+import SafeArea from '../../components/SafeArea';
+import { theme } from '../../infrastructure/index';
+import { BiteShareContext } from '../../BiteShareContext';
+import * as Google from 'expo-google-app-auth';
+import googleImg from '../../../assets/google-logo.png';
+
+const styles = StyleSheet.create({
+  googleLogo: {
+    height: 67,
+    width: 67,
+  }
+});
+
+const GoogleLogin = () => {
+  const navigation = useNavigation();
+  const { state: { authenticated, nickname }, dispatch } = useContext(BiteShareContext);
+
+  const signInAsync = async () => {
+    console.log('GoogleLogin.js 6 | logging in');
+    try {
+      const { type, user } = await Google.logInAsync({
+        iosClientId: '90738707092-9ik3l59ad22hfghgrms9ti5cgn2da675.apps.googleusercontent.com',
+        // androidClientId: '<YOUR_ANDROID_CLIENT_ID>',
+      });
+      if (type === 'success') {
+        console.log('user', user);
+
+        dispatch({ type: 'SET_AUTH', authenticated: true });
+        dispatch({ type: 'SET_EMAIL', email: user.email });
+        dispatch({ type: 'SET_ACCOUNT_HOLDER_NAME', accountHolderName: user.name });
+        dispatch({ type: 'SET_NICKNAME', nickname: user.givenName });
```

Comment:
actually can we use givenName for`accountHolderName` as well? I think some of the food ordering logic updates the DB based on sth like `guest.name === accountHolderName` so if they are different the DB would not be updated correctly

(BTW I just pulled the latest version of this branch and it works great! The above is the only thing I noticed that needs to be changed. 🙌 )